synapse_to_ipynb.py

```diff
@@ -73,7 +73,7 @@ def update_synapse_notebook_from_ipynb(synnb_path: Path, ipynb_path: Path) -> No
 
             # Update the temporary synpase notebook file's 'cells'
             # property with the ipynb src we read earlier
-            with open(tmp_notebook_path, "r+") as f:
+            with os.fdopen(fd, "r+") as f:
```

Comment:
Thanks for contributing! I noticed that I created this bug in commit e82c398dd35b77296040a711851d9984850f418e woops. We can still use the builtin 'open' for closing the fd rather than os.fdopen. If you wouldn't mind making that small change and ill merge it in? Thanks again.

```
with open(fd, "r+") as f:
```

Reply:
Sure! Updated the PR.